### ISSUE TITLE
Not working on IE11 with create-react-app

### ISSUE THREAD

**Issue by r20** [ant-design/ant-design#6190]
### Version
2.10.1

### Environment
IE11

### Reproduction link
[http://codepen.io/benjycui/pen/KgPZrE?editors=001](http://codepen.io/benjycui/pen/KgPZrE?editors=001)

### Steps to reproduce
Open the antd codepen link in IE11. (I didn't fork it, you can use the original)

### What is expected?
An input element for picking the date should appear.

### What is actually happening?
Only a blank screen is seen. 

If you use the IE developer tools you'll see it's due to the startsWith function call which is not supported on IE11.

However, the sample at https://ant.design/components/date-picker/ does work on IE11.

---
I first hit this by using create-react-app and then installing antd and trying the DatePicker.  It worked in Chrome, but not IE11.

**Comment by benjycui** [comment]
https://ant.design/docs/react/getting-started#Compatibility

**Comment by r20** [comment]
@benjycui ,  Thanks!  I remembered reading that antd was supported on IE9+ prior to having the problem.  It might have been from the compatibility page you linked to:

> Ant Design React supports all the modern browsers and IE9+.

So I was expecting it to work.  However, that paragraph is incomplete and misleading if the next section isn't read and/or remembered (which talks about needing polyfills).  I guess I missed or didn't remember that.

After I had the problem I did search for what browsers antd supports and I came across this from https://github.com/ant-design/ant-design:

> Environment Support
>
> Browser: Modern browsers and Internet Explorer 9+
> Server-side Rendering
> Electron

It's missing mention of needing polyfills.

So, documentation could be improved.  But, thanks again for getting me past the problem.
